monitoring/mongodb/dashboard.json

```
@@ -265,1465 +341,5690 @@
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "insertNulls": false,
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "auto",
+            "spanNulls": false,
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "binBps"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 12,
+        "y": 140
       },
-      "type": "graph",
-      "xaxis": {
-        "buckets": null,
-        "mode": "time",
-        "name": null,
-        "show": true,
-        "values": []
+      "id": 1602,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
       },
-      "yaxes": [
+      "targets": [
         {
-          "format": "ops",
-          "label": null,
-          "logBase": 1,
-          "max": null,
-          "min": null,
-          "show": true
+          "datasource": {
+            "type": "prometheus",
+            "uid": "metalk8s-prometheus"
+          },
+          "editorMode": "code",
+          "expr": "rate(mongodb_ss_wt_cache_bytes_written_from_cache{job=~\"zenko/data-db-mongodb-sharded-shard.*-data\", rs_nm=~\"$rs_nm\"}[$__rate_interval])",
+          "instant": false,
+          "legendFormat": "written_from ({{pod}})",
+          "range": true,
+          "refId": "A"
         },
         {
-          "format": "short",
-          "label": null,
-          "logBase": 1,
-          "max": null,
-          "min": null,
-          "show": true
+          "datasource": {
+            "type": "prometheus",
+            "uid": "metalk8s-prometheus"
+          },
+          "editorMode": "code",
+          "expr": "rate(mongodb_ss_wt_cache_bytes_read_into_cache{job=~\"zenko/data-db-mongodb-sharded-shard.*-data\", rs_nm=~\"$rs_nm\"}[$__rate_interval])",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "read_in ({{pod}})",
+          "range": true,
+          "refId": "B"
         }
       ],
-      "yaxis": {
-        "align": false,
-        "alignLevel": null
-      }
+      "title": "WiredTiger Cache I/O",
```

wiredtiger redundant with section
```suggestion
      "title": "Cache I/O",
```